Client/Ecosia/UI/MyImpact/MultiplyImpact.swift

```diff
@@ -243,4 +253,22 @@ final class MultiplyImpact: UIViewController, Themeable {
     @objc private func clear() {
         card?.backgroundColor = .theme.ecosia.impactMultiplyCardBackground
     }
+    
+    private var inviteMessage: String? {
+        guard let link = inviteLink else { return nil }
+        
+        return """
+🌳🔗 \(String.localized(.heyThereWantToPlant))
+\(String.localized(.downloadEcosiaOn))
+\(String.localized(.clickMyInvitation)) \(link)
+\(String.localized(.letsDoThis)) 😊
+
+https://apps.apple.com/us/app/ecosia/id670881887
```

Comment:
@ecotopian As mentioned before, this message is still a work in progress from the product side, so as a placeholder we add the link in plain text, as opposed to rich text, otherwise this will not work on most sharing options.

Reply:
perfect, thanks! 🙏